src/Automation/Automation/Cmdlet/NewAzureAutomationModule.cs

```diff
@@ -44,14 +45,34 @@ public class NewAzureAutomationModule : AzureAutomationBaseCmdlet
         [ValidateNotNullOrEmpty]
         public Uri ContentLinkUri { get; set; }
 
+
+        /// <summary>
+        /// Gets or sets the runbook version type
+        /// </summary>
+        [Parameter(Position = 4,Mandatory = false, HelpMessage = "Runtime Environment of module ")]
```

Comment:
```suggestion
        [Parameter(Mandatory = false, HelpMessage = "Runtime Environment of module ")]
```
Please help confirm whether the `Position` is needed

Reply:
Removed Position parameter